src/Auth.php

```diff
@@ -49,6 +49,19 @@ public static function login_and_get_token( $username, $password ) {
 			return new UserError( __( 'JWT Auth is not configured correctly. Please contact a site administrator.', 'wp-graphql-jwt-authentication' ) );
 		}
 
+		/**
+		 * Add captcha to $_POST for orthers modules
+		 */
+		if( GRAPHQL_JWT_REQUIRE_CAPTCHA ) {
+			if( empty($gRecaptchaResponse) || is_null($gRecaptchaResponse) ) {
+				throw new UserError( __( 'Captcha is required', 'wp-graphql-jwt-authentication' ) );
+			} 
+			
+			$_POST['g-recaptcha-response'] = $gRecaptchaResponse;
```

Comment:
@GautheyValentin While I don't think this is bad practice and has it's uses in testing. It's far from how the `$_POST` global should be used. And possibly should be address further up the funnel so to speak 🤔. Maybe see about getting a official filter in place with the reCaptcha plugin.